sycl/doc/extensions/experimental/sycl_ext_oneapi_non_uniform_groups.asciidoc

```diff
@@ -255,9 +255,9 @@ The following group algorithms support `ballot_group`, `fixed_size_group`,
 
 A ballot-group is a non-contiguous subset of a group, representing a collection
 of all work-items in the group that share the same value of some predicate.
-Ballot-groups are always created in a range of two: the first ballot-group
-contains all work-items where the predicate is true, and the second
-ballot-group contains all work-items where the predicate is false.
+Ballot-groups are always created in a range of two: the one ballot-group
```

Comment:
Note for reviewers: I removed mentions of "first" and "second", because below we explicitly say that work-items where the predicate is true are in ballot-group with local id = 1 and the rest of work-items are in ballot-group with local id = 0.

Which meant that 1 is "first" and 0 is "second". I think here we shouldn't use any ordering-related terms to avoid contradiction with the detailed description of methods below.